lib/src/diktat/java/com/diffplug/spotless/glue/diktat/DiktatFormatterFunc.java

```diff
@@ -46,7 +47,7 @@ public String applyWithFile(String unix, File file) {
 		try {
 			return adapter.format(file, unix, isScript);
 		} catch (DiktatReporting.LintException e) {
-			throw Lint.shortcut(e.lints.stream().map(lint -> Lint.atLine(lint.line, lint.ruleId, lint.detail)).collect(Collectors.toList()));
+			throw Lint.shortcut(e.lints.stream().map(lint -> Lint.atLine(lint.line, lint.ruleId, lint.detail)).collect(toList()));
```

Comment:
counter part to checkstyle verbose madness:

- https://github.com/spring-projects/spring-framework/pull/35861#discussion_r2547430609